lib/jekyll/jekyll-feed.rb

```diff
@@ -47,5 +50,30 @@ def feed_exists?
         File.exist? Jekyll.sanitized_path(@site.source, path)
       end
     end
+
+    # Path to the feed.xslt.xml template file
+    def xslt_source_path
+      File.expand_path "../feed.xslt.xml", File.dirname(__FILE__)
+    end
+
+    def xslt_content
```

Comment:
Same with `content_for_file(file)` or similar. Just trying to DRY things up where we can.